python/gherkin3/token_matcher.py

```diff
@@ -50,13 +91,11 @@ def match_StepLine(self, token):
             title = token.line.get_rest_trimmed(len(keyword))
             self._set_token_matched(token, 'StepLine', title, keyword)
             return True
-
         return False
 
     def match_Comment(self, token):
         if not token.line.startswith('#'):
             return False
-
```

Comment:
No, it not necessary to have a blank line separating the guard clause from the main work of the method, but do you think it is so bad, that it motivates going through the whole method and remove them? I try hard not to ask because you propose changes in code I originally wrote, but from the perspective I try to have when evolving existing code - "be consistent with what is already there" (unless there is a very good reason).